src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -529,12 +529,12 @@ void ASHRAE205ChillerSpecs::oneTimeInit_new(EnergyPlusData &state)
 
     if (this->FlowMode == DataPlant::FlowMode::Constant) {
         // reset flow priority
-        DataPlant::CompData::getPlantComponent(state, this->CWPlantLoc).FlowPriority = DataPlant::LoopFlowStatus::NeedyIfLoopOn;
+        this->CWPlantLoc.comp->FlowPriority = DataPlant::LoopFlowStatus::NeedyIfLoopOn;
     }
 
     else if (this->FlowMode == DataPlant::FlowMode::LeavingSetpointModulated) {
         // reset flow priority
-        DataPlant::CompData::getPlantComponent(state, this->CWPlantLoc).FlowPriority = DataPlant::LoopFlowStatus::NeedyIfLoopOn;
+        this->CWPlantLoc.comp->FlowPriority = DataPlant::LoopFlowStatus::NeedyIfLoopOn;
```

Comment:
This is where you may get a (slight) performance improvement. Accessing a loop directly saves you one array lookup, but accessing a component directly saves you four consecutive lookups.

Reply:
So then getPlantComponent is no longer needed. Why not delete that function?

Reply:
We can delete it once the migration is complete. It's only about 25-30% now.